include/alpaka/core/ApiCudaRt.hpp

```diff
@@ -0,0 +1,374 @@
+#pragma once
```

Comment:
Please add the license header to this file:

```
/* Copyright 2022 Andrea Bocci
 *
 * This file is part of alpaka.
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */
```

We add the license to each header, in cases someone is coping one file out of alpaka the license is still clear.

Reply:
:+1: